docs/api/non_geo_datasets.csv

```diff
@@ -2,6 +2,7 @@ Dataset,Task,Source,# Samples,# Classes,Size (px),Resolution (m),Bands
 `ADVANCE`_,C,"Google Earth, Freesound","5,075",13,512x512,0.5,RGB
 `Benin Cashew Plantations`_,S,Airbus Pléiades,70,6,"1,122x1,186",10,MSI
 `BigEarthNet`_,C,Sentinel-1/2,"590,326",19--43,120x120,10,"SAR, MSI"
+`Cloud Cover Detection`_,S,Radiant MLHub,"11,748","10,980",2,512x512,10,MSI
```

Comment:
```suggestion
`Cloud Cover Detection`_,S,Sentinel-2,"22,728",2,512x512,10,MSI
```
Is it 22,728 in the entire dataset or just in the training data?

Reply:
I'm not clear on what the `# Samples` is supposed to represent. The entire dataset is 22,728, and the train/test split is what was provided in the CSV, as that's how the catalog is split.

Reply:
It's supposed to be the total number, not the split. At least, that's what we've done for other datasets. Not all datasets have a train/test split.